go/test/endtoend/tabletmanager/throttler_topo/throttler_test.go

```diff
@@ -516,15 +510,8 @@ func TestRestoreDefaultQuery(t *testing.T) {
 		_, err := updateThrottlerConfig(true, false, throttlerThreshold.Seconds(), "", false)
 		assert.NoError(t, err)
 	})
-	t.Run("requesting heartbeats", func(t *testing.T) {
-		_ = warmUpHeartbeat(t)
-	})
```

Comment:
There is no need to request heartbeats, since `waitForThrottleCheckStatus` does that anyway.